### DISCUSSION TITLE
Use API of radio-browser.info

### DISCUSSION THREAD

**Discussion by WolfgangDpunkt** [navidrome/navidrome#2748] [Ideas]
To significantly enhance the radio function of navidrome, it would be great to use the API of [radio-browser.info](https://www.radio-browser.info/)
This is a free community-supported directory of radio streams, a kind of Wikipedia for radio stations. Almost certainly the largest of its kind. Very very useful.
The implementation of the API is documented quite extensively: https://api.radio-browser.info/

**Comment by deluan** [comment]
There's a PR already for this integration. I'm going through some big changes in the codebase, but will try to merge it in an upcoming release: https://github.com/navidrome/navidrome/pull/2108

**Reply by WolfgangDpunkt** [reply]
Cool, thank you 🙏
